examples/custom-mods/custom_mods/mods.py

```diff
@@ -102,50 +84,34 @@ def tensorboard_mod(
         logdir_run = os.path.join(logdir, str(msg.metadata.run_id))
 
         node_id = str(msg.metadata.dst_node_id)
-
         server_round = int(msg.metadata.group_id)
 
+        # Let's say we want to measure the time taken to execute the app.
```

Comment:
```suggestion
        # Let's say we want to measure the time taken to run the app.
```
Let's then change this to `run` (ref: https://github.com/adap/flower/pull/5063/files#r1990991693)